lib/wasi/src/runtime/mod.rs

```diff
@@ -14,17 +13,23 @@ use std::{
 use derivative::Derivative;
 use virtual_net::{DynVirtualNetworking, VirtualNetworking};
 
-use crate::runtime::{
-    module_cache::ModuleCache,
-    resolver::{PackageResolver, RegistryResolver},
+use crate::{
+    http::DynHttpClient,
+    os::TtyBridge,
+    runtime::{
+        module_cache::ModuleCache,
+        package_loader::{BuiltinPackageLoader, PackageLoader},
+        resolver::{MultiSourceRegistry, Registry, WapmSource},
+    },
+    WasiTtyState,
 };
 
 /// Represents an implementation of the WASI runtime - by default everything is
 /// unimplemented.
 #[allow(unused_variables)]
 pub trait WasiRuntime
```

Comment:
I think it will make sense to rename this to other thing, since now runners take always a `WasiRuntime` always, which it will obviously don't make sense for Emscripten.

See #3880 (we can address it in a PR apart)